node/node.go

```diff
@@ -369,3 +372,12 @@ func (n *Node) Run(ctx context.Context) {
 func (n *Node) Config() Config {
 	return *n.cfg
 }
+
+func parseVersion(version string, log *utils.ZapLogger) string {
+	semVer, err := semver.NewVersion(version)
+	if err != nil {
+		log.Warnw("Failed to parse Juno version", "version", version, "err", err)
```

Comment:
I was hesitating, but let's not log this error, my arguments:
* we already ignore version string validity in `juno_version` method
* it's mostly static error i.e. it's passed during build time, user has no control over it and we will probably find it before release